packages/contracts/src/contracts/current/protocol/AssetProxy/MixinAuthorizable.sol

```diff
@@ -49,6 +49,10 @@ contract MixinAuthorizable is
             !authorized[target],
             "TARGET_ALREADY_AUTHORIZED"
         );
+        
+        assembly {
```

Comment:
This is never being reset when we remove authorization (also added a task to add tests for this).